src/rgw/rgw_op.h

```diff
@@ -1065,7 +1065,7 @@ class RGWGetACLs : public RGWOp {
 
 class RGWPutACLs : public RGWOp {
 protected:
-  size_t len;
+  int len;
```

Comment:
well this was to call `rgw_rest_read_all_input` which expected an int ptr, which returns a -ve number if getting the data failed I believe